### PR TITLE
Add users table to complete PT-55

### PR DESCRIPTION
**Pull request by michardy**

### CODE REVIEW

File: app/db.py [hunk 1]
```diff
@@ -16,6 +16,11 @@ class Pageloads(Base):
     page_end = Column(DateTime, nullable=True)
 
 
+class Users(Base):
+    __tablename__ = 'users'
+    name = Column(String(), primary_key=True)
```
Comment (jmun10) [inline app/db.py:26]: do you need () after string if you are not defining a length?

**Review by hli086 — APPROVED** [review]
lgtm

File: app/db.py (hunk 1 above)
Comment (michardy) [reply app/db.py:26]: No which means the decision falls to our coding style.  I need to figure out what PEP-8 says

Comment (michardy) [reply app/db.py:26]: PEP-8 does not say what to do. We might want to standardize this ourselves

Comment (jmun10) [reply app/db.py:26]: @michardy im ok with either, your call

**Review by jmun10 — APPROVED** [review]
lgtm

**Pull request merged**